round/schema.graphql

```diff
@@ -14,7 +14,7 @@ type MetaPtr @entity {
 # == Allo Settings == #
 #######################
 
-type AlloSettings @entity {
```

Comment:
Hmm this might break things on grants-stack
@gravityblast / @lmcorbalan was this verified with grants-stack UI ?